launch/launch/event_handlers/event_named.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for event_named() function."""
+
+from typing import Callable
+from typing import Text
+
+from ..event import Event
+
+
+def event_named(name: Text) -> Callable[[Event], bool]:
```

Comment:
this function doesn't seem to be used anywhere. not necessarily an issue, but wanted to point it out in case it should be!

Reply:
I used it before, but swapped in all cases to using `is_a_subclass` since this is Python, but I believe (hope) it will be useful when transitioning to a markup language where you don't have handles to actions or events, and need to reference them by name.